### PR TITLE
fix: install epel repo in RHEL 7 using static repo file 

### PR DESCRIPTION
**Pull request by gflorinskaya**
**What problem does this PR solve?**:
We are facing flakiness when installing the epel repo 

**Which issue(s) does this PR fix?**:

* https://jira.d2iq.com/browse/D2IQ-NUMBER

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Retry epel yum repo install up to 30s
```

### CODE REVIEW

File: ansible/roles/repo/tasks/redhat.yaml [hunk 1]
```diff
@@ -124,5 +126,5 @@
     gpgcheck: true
   register: konvoy_repo_installation_rpm
   until: konvoy_repo_installation_rpm is success
-  retries: 3
-  delay: 3
```
Comment (dkoshkin) [inline ansible/roles/repo/tasks/redhat.yaml:128]: Was this change intentional?

Comment (gflorinskaya) [reply ansible/roles/repo/tasks/redhat.yaml:128]: Yes, it was intentional, we are trying to keep it close to the PR that @dlipovetsky is working on. 
Should I update the commit message?

**Comment by supershal** [comment]
This rpm URL is not reliable. it can fail after multiple attempts too. 
Instead should we copy the repo files and public GPG keys on host?
```
[root@b12318a4f6f9 /]# cat /etc/yum.repos.d/epel.repo
[epel]
name=Extra Packages for Enterprise Linux 7 - $basearch
#baseurl=http://download.fedoraproject.org/pub/epel/7/$basearch
metalink=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
failovermethod=priority
enabled=1
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7

[epel-debuginfo]
name=Extra Packages for Enterprise Linux 7 - $basearch - Debug
#baseurl=http://download.fedoraproject.org/pub/epel/7/$basearch/debug
metalink=https://mirrors.fedoraproject.org/metalink?repo=epel-debug-7&arch=$basearch
failovermethod=priority
enabled=0
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
gpgcheck=1

[epel-source]
name=Extra Packages for Enterprise Linux 7 - $basearch - Source
#baseurl=http://download.fedoraproject.org/pub/epel/7/SRPMS
metalink=https://mirrors.fedoraproject.org/metalink?repo=epel-source-7&arch=$basearch
failovermethod=priority
enabled=0
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
gpgcheck=1
```

and
```
[root@b12318a4f6f9 /]# ls etc/pki/rpm-gpg/
RPM-GPG-KEY-CentOS-7  RPM-GPG-KEY-CentOS-Debug-7  RPM-GPG-KEY-CentOS-Testing-7  RPM-GPG-KEY-EPEL-7
```

**Review by faiq — APPROVED** [review]

**Review by supershal — APPROVED** [review]

**Comment by supershal** [comment]
created a stacked PR. https://github.com/mesosphere/konvoy-image-builder/pull/782
